templates/4b_normalize_image_data.ipynb

```diff
@@ -37,7 +37,8 @@
    "source": [
```

Comment:
<p>Kind a crowded bullet point. I think we could tweak the explanation a bit.</p><p> </p><p>- <code>tuning_curve_file</code> should point to a tuning curve contained in <code>toffy/tuning_curves</code> (<code>P9_450_norm_func.json</code>, <code>P9_1300_norm_func.json</code>, or <code>P9_2600_norm_func.json</code>)</p><p> </p><p>Include the ionpath default specification in bold underneath.</p><p>**<code>P9_2600_norm_func.json</code> is the curve that should be used when running the instrument with Ionpath's default settings.** <code>P9_450_norm_func.json</code> and <code>P9_1300_norm_func.json</code> are curves with Angelo Lab-specific parameters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>